earn/src/data/Uniboost.ts

```diff
@@ -142,7 +142,12 @@ export async function fetchBoostBorrowersList(
   const boostNftContract = new ethers.Contract(ALOE_II_BOOST_NFT_ADDRESS[chainId], BoostNftAbi, provider);
 
   // Figure out how many Boost NFTs the user has
-  const numBoostNfts: number = (await boostNftContract.balanceOf(userAddress)).toNumber();
+  let numBoostNfts: number = 0;
+  try {
+    numBoostNfts = (await boostNftContract.balanceOf(userAddress)).toNumber();
+  } catch (e) {
+    return { borrowers: [], tokenIds: [] };
+  }
```

Comment:
Was running into some errors on initial load since the chain is still the default (optimism), looking into this soon.

Reply:
This is a complex fix that may take some time. I would remove this after.